.github/actions/verify-submodule-pull-request/functions.ps1

```diff
@@ -0,0 +1,10 @@
+<#
+.Synopsis
+Fails the current step with the provided message. Same as "core.setFailed()" from the official toolkit
+(https://github.com/actions/toolkit).
+#>
+function Set-Failed([Parameter(Mandatory = $True)] [string] $Message)
+{
+    Write-Output "::error::$Message"
```

Comment:
Add a note about these special messages to https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows.